tools/pipelines/templates/build-npm-package.yml

```diff
@@ -139,6 +143,10 @@ variables:
   - name: toolAbsolutePath
     value:  $(Build.SourcesDirectory)/tools/telemetry-generator
     readonly: true
+  - name: testWorkspace
+    value: $(Pipeline.Workspace)/test
+  - name: loggerPackage
+    value: '@ff-internal/aria-logger'
```

Comment:
I'd say skip this one altogether, since it's the default of the telemetry setup template and that's the only place we used it.